src/Widgets/CardWidget/CardWidgetComponent.tsx

```diff
@@ -99,3 +84,40 @@ const LinkOrNotTag = connect(
     )
   },
 )
+
+const ImageOrVideo = connect(function ImageOrVideo({
```

Comment:
* [ ] FYI Now we have three instances of ImageOrVideo I think it's a good time to DRY up.